SPECS/efibootmgr/efibootmgr.spec

```diff
@@ -33,6 +33,9 @@ gzip -9 %{buildroot}%{_mandir}/man8/efibootdump.8
 %{_sbindir}/*
 %{_mandir}/man8/*
 %changelog
+* Wed Sep 20 2023 Jon Slobodzian <joslobo@microsoft.com> - 16-4
```

Comment:
can you add a blank space before %changelog

Reply:
Fixed